src/components/gallery/EventsGallery.tsx

```diff
@@ -23,137 +25,165 @@ const EventsGallery = () => {
 			</div>
 			<div className="grid grid-cols-1 gap-4 lg:grid-cols-2 lg:gap-16">
```

Comment:
maybe we could `md:grid-cols-2` instead of `lg:grid-cols-2` since we have enough empty space in medium breakpoint. We could also decrease the `gap`.

Reply:
I added some modification to this section. You can check it if you have some time.

Reply:
Nice 👍🏾